src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/updatereasonableadjustment/UpdateReasonableAdjustmentAboutToSubmitHandler.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.updatereasonableadjustment;
+
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.isNoOrNull;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.isYes;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+
+@Component
+@Slf4j
+public class UpdateReasonableAdjustmentAboutToSubmitHandler implements PreSubmitCallbackHandler<SscsCaseData> {
+
+    @Override
+    public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callback) {
+        return callbackType == CallbackType.ABOUT_TO_SUBMIT
+            && callback.getEvent() == EventType.UPDATE_REASONABLE_ADJUSTMENT
+            && nonNull(callback.getCaseDetails())
+            && nonNull(callback.getCaseDetails().getCaseData());
+    }
+
+    @Override
+    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback, String userAuthorisation) {
+        if (!canHandle(callbackType, callback)) {
+            throw new IllegalStateException("Cannot handle callback");
+        }
+
+        SscsCaseData sscsCaseData = callback.getCaseDetails().getCaseData();
+
+        if (isYes(sscsCaseData.getAppeal().getAppellant().getIsAppointee()) || nonNull(sscsCaseData.getReasonableAdjustment().getAppellant()) && isNoOrNull(sscsCaseData.getReasonableAdjustment().getAppellant().getWantsReasonableAdjustment())) {
+            sscsCaseData.getReasonableAdjustment().setAppellant(null);
+        }
+
+        if (isNoOrNull(sscsCaseData.getAppeal().getAppellant().getIsAppointee()) || nonNull(sscsCaseData.getReasonableAdjustment().getAppointee()) && isNoOrNull(sscsCaseData.getReasonableAdjustment().getAppointee().getWantsReasonableAdjustment())) {
```

Comment:
Similar to comment above - this logic will always hit if there is an appointee on the case. 
See here: https://introcs.cs.princeton.edu/java/11precedence/ 
AND statements take precedence over OR statements, so think you are missing some brackets

Reply:
It says if there is no appointee Or the appointee does not want reasonable adjustments, set the appointee's alternative letter format to null.

Reply:
ok